app/bootstrap.php

```diff
@@ -23,12 +23,13 @@
 
 $app = new Application();
 
-$configFile = __DIR__ . '/../config.json';
-if (! file_exists($configFile)) {
-    $configFile = __DIR__ . '/../config.json.dist';
-}
+$env = new \Dotenv\Dotenv(__DIR__ . '/../');
+$env->load();
```

Comment:
Let's add some validation here? https://github.com/vlucas/phpdotenv#requiring-variables-to-be-set